src/components/terminal/terminal.tsx

```diff
@@ -0,0 +1,267 @@
+'use client'
+import { useRouter } from 'next/navigation'
+import { useEffect, useRef } from 'react'
+import { Terminal as XtermTerminal } from '@xterm/xterm'
+import { FitAddon } from '@xterm/addon-fit'
+import { WebLinksAddon } from '@xterm/addon-web-links'
+import '@xterm/xterm/css/xterm.css'
+import type { ITerminalInitOnlyOptions, ITerminalOptions } from '@xterm/xterm'
+
+const XtermTerminalOptions: ITerminalOptions & ITerminalInitOnlyOptions = {
+  fontSize: 14,
+  fontFamily: 'Geist Mono, monospace',
+  // TODO: does this work?
+  macOptionIsMeta: true,
+  cursorBlink: true,
```

Comment:
`cursorBlink` don't work

Reply:
It's bug https://github.com/xtermjs/xterm.js/issues/4987
WebGL addon does fix this, but cannot test the inner contents.